packages/@aws-cdk/aws-apigatewayv2-integrations/README.md

```diff
@@ -146,3 +148,28 @@ const httpEndpoint = new HttpApi(stack, 'HttpProxyPrivateApi', {
   }),
 });
 ```
+
+## WebSocket APIs
+
+WebSocket integrations connect a route to backend resources. Currently supported integrations include: LambdaWebSocketIntegration
+
+### Lambda WebSocket Integration
+
+Lambda integrations enable integrating a WebSocket API route with a Lambda function. When a client connects/disconnects or sends message specific to a route, the API Gateway service forwards the request to the Lambda function
+
+The API Gateway service will invoke the lambda function with an event payload of a specific format.
```

Comment:
Any details we can link to on what the lambda event request will contain and what the schema of the response should be.

Reply:
I looked around the API reference and developer docs but was not able to find anything unfortunately. We can add it in later changes once we have one.